src/systemctl/systemctl-compat-halt.c

```diff
@@ -76,6 +76,8 @@ int halt_parse_argv(int argc, char *argv[]) {
         assert(argc >= 0);
         assert(argv);
 
+        /* called in sysvinit system as last command in
+         * shutdown/reboot so this is always forceful */
```

Comment:
btw, we usually break lines at 109 chars, see CODING_STYLE, .editorconfig and so on.